src/DynamoCore/Graph/Nodes/Attributes.cs

```diff
@@ -18,20 +18,40 @@ public NodeNameAttribute(string elementName)
         public string Name { get; set; }
     }
 
+    /// <summary>
+    ///     The NodeCategoryAttribute attribute allows the node implementer
+    ///     to define in which category node will appear, regardless its namespace. 
+    /// </summary>
     [AttributeUsage(AttributeTargets.All)]
     public class NodeCategoryAttribute : Attribute
     {
+        /// <summary>
+        /// Creates NodeCategoryAttribute.
+        /// </summary>
+        /// <param name="category">Full name of the category. E.g. Core.List.Create</param>
         public NodeCategoryAttribute(string category)
         {
             ElementCategory = category;
         }
 
+        /// <summary>
+        /// Full name of the category. E.g. Core.List.Create
+        /// </summary>
         public string ElementCategory { get; set; }
     }
 
+    /// <summary>
+    ///     The NodeCategoryAttribute attribute allows the node implementer
```

Comment:
The NodeSearchTagsAttribute attribute allows the node implementer to  define search tags